Update working-with-boards.rst (#5325)
mattermost/docs: source/boards/working-with-boards.rst

Changing views
--------------

You can change boards views to adjust how your cards are represented. To add a new view to a board, select the menu next to the current view name. Scroll down and select **+ Add view**, then select the new visualization you'd like to use.

Board view
~~~~~~~~~~

This is a vertical list view where cards are sorted by columns. The column names are editable and you can drag cards between columns.

Calendar view
~~~~~~~~~~~~~

To use this view, cards need to have the **Date** property added.

> If cards don't have a **Date** property, they'll be sorted and displayed by the card creation date (default). These cards can't be moved around the board until a **Date** property is added. This property can be a due date, milestone, and so on.

Author: ```suggestion
If cards don't have a custom **Date** property, they'll be sorted and displayed by the card creation date (default). These cards can't be moved around the board until a custom **Date** property is added.
```
